src/components/PostList.js

```diff
@@ -1,9 +1,15 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { fetchPosts } from '../actions';
 
 class PostList extends React.Component {
+    componentDidMount() {
+        this.props.fetchPosts();
+    }
+    
     render() {
         return <div>Post List</div>;
     }
 }
 
-export default PostList;
+export default connect(null, { fetchPosts })(PostList);
```

Comment:
using connect without mapStateToProps but adding it the imported fetchPosts Action